lib/timezone/lookup/google.rb

```diff
@@ -10,6 +10,11 @@ module Timezone
   module Lookup
     # @!visibility private
     class Google < ::Timezone::Lookup::Basic
+      # indicates that no time zone data could be found for the specified
+      # position or time. Confirm that the request is for a location on land,
+      # and not over water.
```

Comment:
This might be clearer as "Indicates that no time zone data could be found for the specified <lat, lng>. This can occur if the query is incomplete or ambiguous."

https://maps-apis.googleblog.com/2016/11/address-geocoding-in-google-maps-apis.html